resources/glossary/url_map.json

```diff
@@ -87,13 +87,13 @@
     "title": "Data Types",
     "description": "Data types describe a space of possible values through the specification of constraints",
     "slug": "data-types",
-    "tags": ["Graphs", "Standards"],
+    "tags": ["Graphs", "Standards"]
   },
   {
-   "title": "Datasets",
-   "description": "Datasets are collections of numbers of words, generally centered around a single topic or subject.",
-   "slug": "datasets",
-   "tags": ["Business Intelligence", "Data Science"],
+    "title": "Datasets",
+    "description": "Datasets are collections of numbers of words, generally centered around a single topic or subject.",
```

Comment:
Should this be numbers _or_ words? @nonparibus

Reply:
Yes :(

Reply:
```suggestion
    "description": "Datasets are collections of numbers or words, generally centered around a single topic or subject.",
```